audit/projects/spiffxp-boskos-project-01/description.json

```diff
@@ -0,0 +1,11 @@
+{
+  "createTime": "2020-04-26T17:09:06.114Z",
+  "lifecycleState": "ACTIVE",
+  "name": "spiffxp-boskos-project-01",
```

Comment:
Let's not forget to clean these up :)

Reply:
this is exactly why I put my name on them